frontend/src/metabase/query_builder/containers/QueryBuilder.jsx

```diff
@@ -245,6 +248,12 @@ function QueryBuilder(props) {
     clearTimeout(timeout.current);
   });
 
+  useEffect(() => {
+    if (question && question.isSaved() && question.hasBreakoutByDateTime()) {
+      fetchTimelines(question.card());
```

Comment:
It seems like `fetchTimelines`  loads events by a card id only for one specific card at a time. Why did you choose to load timeline events for a card using a separate request instead of extending `card` api resource? I assume showing these events on dashboards will require performing one more request for each card. Was it due to potential BE performance issues?